tasks/auto-generate-skus.json

```diff
@@ -1,16 +1,16 @@
 {
   "name": "Auto-generate SKUs",
   "options": {
-    "skip_variants_that_already_have_skus__boolean": false,
+    "skip_variants_that_already_have_skus__boolean": true,
```

Comment:
This is a significant opinion change, far as defaults go, and also I think I agree that this is a better value. :) Better to do less by default.